### PR TITLE
MM-17047 - Making post text focusable

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-17047 - Making post text focusable

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-17047

### CODE REVIEW

File: components/post_view/post_message_view/post_message_view.jsx [hunk 1]
```diff
@@ -168,6 +168,8 @@ export default class PostMessageView extends React.PureComponent {
                 text={message}
             >
                 <div
+                    aria-readonly='true'
+                    tabIndex='0'
```
Comment (devinbinnie) [inline components/post_view/post_message_view/post_message_view.jsx]: Might be a dumb question, but wouldn't this apply globally throughout the page? How does it work with menus and when we want to tab through them?

Comment (asaadmahmood) [reply components/post_view/post_message_view/post_message_view.jsx]: `tabIndex` just makes the element focusable, `0` does not place the element in a specific flow order, the flow order is inherited by the element's position on the page.

**Comment by cpanato** [comment]
a bit mess the test servers, will improve a bit the flow later on

**Comment by esethna** [comment]
Good to merge, thanks Asaad

**Pull request merged**